deploy/registration-service/registration-service.yaml

```
@@ -222,13 +223,13 @@ objects:
       namespace: ${NAMESPACE}
       labels:
         provider: codeready-toolchain
-        run: registration-service
+        run: proxy-metrics
     spec:
       ports:
         - name: "8082"
```

```suggestion
        - name: metrics
```
we can name the port on the service with `metrics`, too

---

wouldn't it have to be "metrics" then?
But I like having the name of the port and the target-port different, less confusing imo. Like in this example - https://kubernetes.io/docs/concepts/services-networking/service/#field-spec-ports - I wouldn't have understood this example if the service port name and target port name were the same. 
Not saying its the right thing to do, but its just me and my brain.

---

done in [4367241](https://github.com/codeready-toolchain/host-operator/pull/890/commits/4367241f5db752741c30ff757e0088549be0cffd)

---

done in [4367241](https://github.com/codeready-toolchain/host-operator/pull/890/commits/4367241f5db752741c30ff757e0088549be0cffd)